new_matrixfit.Rmd

```diff
@@ -0,0 +1,71 @@
+```{r setup}
```

Comment:
shouldn't this be in the vignettes or something?

Reply:
In principle yes, but that is just the exploratory notebook to test the functions. I did not consider it a vignette that shows off the package, so I did not want to include it there yet.